common/tables/table-en.html

```diff
@@ -0,0 +1,622 @@
+			</tr>
+		</tbody>
+	</table>
+</div>
+<div class="table-responsive">
+	<table class="table table-bordered">
+		<thead>
+			<tr>
+				<th>Number</th>
+				<th>Table heading</th>
+				<th>Table heading</th>
+				<th>Table heading</th>
+				<th>Table heading</th>
+				<th>Table heading</th>
+				<th>Table heading</th>
+			</tr>
+		</thead>
+		<tbody>
+			<tr>
+				<th>1</th>
+				<td>Table cell</td>
+				<td>Table cell</td>
+				<td>Table cell</td>
+				<td>Table cell</td>
+				<td>Table cell</td>
+				<td>Table cell</td>
+			</tr>
+			<tr>
+				<th>2</th>
+				<td>Table cell</td>
+				<td>Table cell</td>
+				<td>Table cell</td>
+				<td>Table cell</td>
+				<td>Table cell</td>
+				<td>Table cell</td>
+			</tr>
+			<tr>
+				<th>3</th>
+				<td>Table cell</td>
+				<td>Table cell</td>
+				<td>Table cell</td>
+				<td>Table cell</td>
+				<td>Table cell</td>
+				<td>Table cell</td>
+			</tr>
+		</tbody>
+	</table>
+</div>
+<pre><code>&lt;div class="table-responsive"&gt;
+&lt;table class="table"&gt;
+...
+&lt;/table&gt;
+&lt;/div&gt;</code></pre>
+
+<h2>Table Grids <code>.tbl-gridify</code></h2>
+<table class="table tbl-gridify">
+	<thead>
+		<tr>
+			<th>Number</th>
+			<th>Column heading</th>
+			<th>Column heading</th>
+			<th>Column heading</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr class="active">
+			<th>1</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+		<tr>
+			<th>2</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+		<tr class="success">
+			<th>3</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+		<tr>
+			<th>4</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+		<tr class="info">
+			<th>5</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+		<tr>
+			<th>6</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+		<tr class="warning">
+			<th>7</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+		<tr>
+			<th>8</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+		<tr class="danger">
+			<th>9</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+	</tbody>
+</table>
+<pre><code>&lt;table class="table tbl-gridify"&gt;
+...
+&lt;/table&gt;</code></pre>
+
+<h2> Tables within panels </h2>
+<p>Full-width content:</p>
+	<div class="panel panel-default">
+		<table class="table">
+		<caption class="wb-inv">Caption text</caption>
+		<thead>
+			<tr>
+				<th>Table header</th>
+				<th>Table header</th>
+			</tr>
+		</thead>
+		<tbody>
+			<tr>
+				<td>Table data</td>
+				<td>Table data</td>
+			</tr>
+			<tr>
+				<td>Table data</td>
+				<td>Table data</td>
+			</tr>
+		</tbody>
+	</table>
+</div>
```

Comment:
Fixing the indentation and adjusting the code to illustrate a full demo of a table used in a conforming and complete panel where there is a distinction between the table caption and the panel header.

Note: those example are coming from the style guide but may be at the time of writing the style guide the author didn't know how to use the CSS border helper along with the corner radius option. 


```suggestion
<section class="panel panel-default">
	<header class="panel=heading">
		<h3>Panel title</h3>
	</header>
	<div class="panel-body">
		<p>Description of the topic discussed in this panel</p>
		<p>Lorem ipsum...</p> 
		<table class="table">
			<caption>Table caption text</caption>
			<thead>
				<tr>
					<th>Table header</th>
					<th>Table header</th>
				</tr>
			</thead>
			<tbody>
				<tr>
					<td>Table data</td>
					<td>Table data</td>
				</tr>
				<tr>
					<td>Table data</td>
					<td>Table data</td>
				</tr>
			</tbody>
		</table>
	</div>
	<footer class="panel-footer">
		<p>Panel footer</p>
	</footer>
</section>
```

Note: Adapt the code sample accordingly.